mne/surface.py

```diff
@@ -1978,8 +1978,11 @@ def _marching_cubes(image, level, smooth=0, fill_hole_size=None, use_flying_edge
         polydata = geometry.GetOutput()
         rr = vtk_to_numpy(polydata.GetPoints().GetData())
         tris = vtk_to_numpy(polydata.GetPolys().GetConnectivityArray()).reshape(-1, 3)
-        rr = np.ascontiguousarray(rr[:, ::-1])
-        tris = np.ascontiguousarray(tris[:, ::-1])
+        if order == "C":
```

Comment:
Then you don't need this conditional and should be able to revert this change (since the `::-1` is the `fliplr` equiv anyway)